e2e-test/src/test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,50 @@
+projectId=cdf-athena
+dataset=test_automation
+csvFormat=csv
+
+## FILE-PLUGIN-PROPERTIES-START
+incorrectFilePath=invalidPath
+csvFile=testdata/file/CSV_TEST.csv
+csvFileSchema=[{"key":"EmployeeDepartment","value":"string"},{"key":"Employeename","value":"string"},\
+  {"key":"Salary","value":"int"},{"key":"workhours","value":"int"}]
+csvFileOutputSchema={ "type": "record", "name": "text", "fields": [ \
+  { "name": "EmployeeDepartment", "type": "string" }, { "name": "Employeename", "type": "string" }, \
+  { "name": "Salary", "type": "int" }, { "name": "workhours", "type": "int" } ] }
+csvAllDataTypeFile=testdata/file/CSV_DATATYPE_TEST_1.csv
+csvAllDataTypeFileSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
```

Comment:
Make sure we test all supported data types. Looks like we're only testing int, string, boolean, double

Reply:
Updated.